### PR TITLE
Add Firefox

### PR DESCRIPTION
**Pull request by danielopez26**

### CODE REVIEW

**Review by aperdomob — CHANGES_REQUESTED** [review]
Do you execute both browsers in your CI?

**Comment by danielopez26** [comment]
Yes, I have defined both controllers and run each of them

File: src/main/java/GoogleSearch.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import drivers.Drivers;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.junit.jupiter.api.*;
+
+import java.util.concurrent.TimeUnit;
+
+public class GoogleSearch {
```
Comment (aperdomob) [inline src/main/java/GoogleSearch.java:9]: you don't need this class

File: src/main/java/drivers/Drivers.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package drivers;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class Drivers {
+
+    WebDriver driver;
```
Comment (aperdomob) [inline src/main/java/drivers/Drivers.java:10]: don't use global variable

File: src/main/java/drivers/Drivers.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+    public String BROWSER = System.getProperty("BROWSER", "Firefox");
```
Comment (aperdomob) [inline src/main/java/drivers/Drivers.java:12]: this class should not have this logic, also use camel case to name of variables

**Review by aperdomob — CHANGES_REQUESTED** [review]

File: src/main/java/drivers/Drivers.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+public class Drivers {
+
+    public WebDriver strategyDriver(){
+
+        String browser = System.getProperty("browser", "Firefox");
+        switch (browser){
+            case "Chrome":
+                WebDriverManager.chromedriver().setup();
+                WebDriver chromeDriver = new ChromeDriver();
+                return chromeDriver;
```
Comment (aperdomob) [inline src/main/java/drivers/Drivers.java:17]: Don't create the `WebDriver chromeDriver` only return the new instance

```java
return new ChromeDriver();
```

File: src/main/java/drivers/Drivers.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+            case "Firefox":
+                WebDriverManager.firefoxdriver().setup();
+                WebDriver firefoxDriver = new FirefoxDriver();
+                return firefoxDriver;
```
Comment (aperdomob) [inline src/main/java/drivers/Drivers.java:21]: same comment here

**Review by aperdomob — CHANGES_REQUESTED** [review]

**Review by aperdomob — APPROVED** [review]

**Pull request merged**